.github/workflows/run-examples.yml

```diff
@@ -12,13 +12,46 @@ jobs:
     name: Examples
     runs-on: ubuntu-latest
     steps:
-      - name: Check out code
-        uses: actions/checkout@v2
-      - name: Install dependencies
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.20.7
+        id: go
+
+      - name: Checkout mx-chain-go
+        uses: actions/checkout@v4
+        with:
+          repository: 'multiversx/mx-chain-go'
+          ref: 'feat/chain-go-sdk'
+          path: 'mx-chain-go'
+
+      - name: Checkout mx-chain-simulator-go
+        uses: actions/checkout@v4
+        with:
+          path: 'mx-chain-simulator-go'
+
+      - name: Get dependencies
         run: |
-          python3 -m pip install --upgrade pip
-          pip3 install -r examples/requirements.txt
-          pip3 install pytest
+          cd mx-chain-simulator-go
+          go get -v -t -d ./...
+          if [ -f Gopkg.toml ]; then
+              curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+              dep ensure
+          fi
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+
+      - name: Install Python Dependencies
+        run: |
+          pip install -r mx-chain-simulator-go/examples/requirements.txt
+
       - name: Run examples
         run: |
-          make run-examples
```

Comment:
This `make` command is using docker and we don't have docker for sovereign chain simulator